README.md

```diff
@@ -21,20 +35,124 @@ To run CDE in a box docker-compose you need graphDB triple store free edition. F
 ```sh
 mv graphdb-free-{version}-dist.zip cde-in-box/graph-db
 ```
+**Step 4:** Execute:
 
-**Step 4:** If your `graphdb version` is different from `9.7.0` then change the version number of graph DB in the docker-compose file.
+```
+docker volume create graphdb-data
+```
 
-```sh
-graph_db:
+**Step 5:** If your `graphdb version` is different from `9.7.0` then change the version number of graph DB in the docker-compose file.
+
+```
+version: '3'
+
+services:
+  #  Graph DB service. BEFORE you start running docker-compose file please make sure that you have downloaded free edition of graphDB zip files	
+  graphdb:
+    image: graph-db:9.7.0
     build:
       context: ./graph-db
       dockerfile: Dockerfile        
       args:
         version: 9.7.0
+        edition: free
+    restart: always
+    hostname: graphdb
+    ports:
+      - 7200:7200
+    volumes:
+      - graphdb-data:/opt/graphdb/home
+    
+  #  This service create's `cde` and `fdp` repositories in graphdb	
+  graph_db_repo_manager:
+    build: ./graph-db-repo-manager
+    depends_on:
+      - graphdb
+    environment:
+      - "GRAPH_DB_URL=http://graphdb:7200"
+
+volumes:
+  graphdb-data:
+    external: true
+
+```
+
+
+**Step 6:** Then execute a docker-compose up.   Let it go through the initialization process, and then stop everything.
+
+```sh
+docker-compose up -d
+```
+
+You now have a docker container 'graph-db:9.7.0' that we will call in this new docker-compose.yml file:
+
+```
+version: "2.0"
+services:
+  
+  graphdb:
```

Comment:
Since the other docker-compose file is already creating a graphDB service maybe you can reuse that service right? I will create a network like [this](https://github.com/kburger/twoc-fdp-guide/blob/develop/docker-compose.yml#L8) in other docker-compose file